### PR TITLE
Update outdated files in the dev-1.18-ko.11 branch

### LINKED ISSUE #23532: Outdated files in the dev-1.18-ko.11 branch
# This is a Bug Report
## Problem
Outdated files in the dev-1.18-ko.11 branch.

### 70 files to be modified 
1. [ ] content/en/_index.html 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/_index.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/concepts/architecture/controller.md 1 file changed, 4 insertions(+), 5 deletions(-)
1. [ ] content/en/docs/concepts/architecture/nodes.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/cluster-administration/addons.md 1 file changed, 1 insertion(+), 3 deletions(-)
1. [ ] content/en/docs/concepts/cluster-administration/cloud-providers.md 1 file changed, 5 insertions(+), 76 deletions(-)
1. [ ] content/en/docs/concepts/configuration/manage-resources-containers.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/configuration/resource-bin-packing.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/configuration/secret.md 1 file changed, 4 insertions(+)
1. [ ] content/en/docs/concepts/containers/container-lifecycle-hooks.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/containers/images.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/extend-kubernetes/extend-cluster.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/extend-kubernetes/operator.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/storage/persistent-volumes.md 1 file changed, 10 insertions(+)
1. [ ] content/en/docs/concepts/storage/volume-snapshots.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/workloads/controllers/daemonset.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/controllers/deployment.md 1 file changed, 7 insertions(+), 7 deletions(-)
1. [ ] content/en/docs/concepts/workloads/controllers/garbage-collection.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/controllers/job.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/controllers/replicaset.md 1 file changed, 4 insertions(+), 3 deletions(-)
1. [ ] content/en/docs/concepts/workloads/controllers/replicationcontroller.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/controllers/statefulset.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/pods/_index.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/pods/pod-lifecycle.md 1 file changed, 7 insertions(+), 8 deletions(-)
1. [ ] content/en/docs/reference/_index.md 1 file changed, 4 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/reference/command-line-tools-reference/_index.md 1 file changed, 1 deletion(-)
1. [ ] content/en/docs/reference/command-line-tools-reference/feature-gates.md 1 file changed, 8 insertions(+), 5 deletions(-)
1. [ ] content/en/docs/reference/glossary/kube-apiserver.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/reference/glossary/managed-service.md 1 file changed, 6 insertions(+), 1 deletion(-)
1. [ ] content/en/docs/reference/issues-security/_index.md 1 file changed, 1 deletion(-)
1. [ ] content/en/docs/reference/kubectl/cheatsheet.md 1 file changed, 25 insertions(+), 28 deletions(-)
1. [ ] content/en/docs/reference/kubectl/overview.md 1 file changed, 18 insertions(+), 12 deletions(-)
1. [ ] content/en/docs/reference/setup-tools/_index.md 1 file changed, 1 deletion(-)
1. [ ] content/en/docs/reference/setup-tools/kubeadm/_index.md 1 file changed, 26 insertions(+), 1 deletion(-)
1. [ ] content/en/docs/reference/using-api/_index.md 1 file changed, 1 deletion(-)
1. [ ] content/en/docs/reference/using-api/api-overview.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/reference/using-api/client-libraries.md 1 file changed, 3 insertions(+), 3 deletions(-)
1. [ ] content/en/docs/setup/production-environment/container-runtimes.md 1 file changed, 82 insertions(+), 40 deletions(-)
1. [ ] content/en/docs/tasks/_index.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/access-application-cluster/access-cluster.md 1 file changed, 9 insertions(+), 9 deletions(-)
1. [ ] content/en/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md 1 file changed, 12 insertions(+), 26 deletions(-)
1. [ ] content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md 1 file changed, 114 insertions(+), 38 deletions(-)
1. [ ] content/en/docs/tasks/administer-cluster/access-cluster-api.md 1 file changed, 7 insertions(+), 10 deletions(-)
1. [ ] content/en/docs/tasks/administer-cluster/access-cluster-services.md 1 file changed, 3 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/tasks/administer-cluster/cluster-management.md 1 file changed, 18 insertions(+), 10 deletions(-)
1. [ ] content/en/docs/tasks/administer-cluster/dns-custom-nameservers.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md 1 file changed, 1 insertion(+), 5 deletions(-)
1. [ ] content/en/docs/tasks/administer-cluster/network-policy-provider/romana-network-policy.md 1 file changed, 4 insertions(+), 12 deletions(-)
1. [ ] content/en/docs/tasks/administer-cluster/network-policy-provider/weave-network-policy.md 1 file changed, 10 insertions(+), 14 deletions(-)
1. [ ] content/en/docs/tasks/configure-pod-container/quality-service-pod.md 1 file changed, 6 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/tasks/debug-application-cluster/debug-init-containers.md 1 file changed, 3 insertions(+), 7 deletions(-)
1. [ ] content/en/docs/tasks/debug-application-cluster/logging-elasticsearch-kibana.md 1 file changed, 4 insertions(+), 3 deletions(-)
1. [ ] content/en/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md 1 file changed, 16 insertions(+), 8 deletions(-)
1. [ ] content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md 1 file changed, 9 insertions(+), 6 deletions(-)
1. [ ] content/en/docs/tasks/inject-data-application/define-environment-variable-container.md 1 file changed, 1 insertion(+), 11 deletions(-)
1. [ ] content/en/docs/tasks/manage-daemon/update-daemon-set.md 1 file changed, 1 insertion(+), 8 deletions(-)
1. [ ] content/en/docs/tasks/manage-kubernetes-objects/declarative-config.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/manage-kubernetes-objects/imperative-command.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/manage-kubernetes-objects/imperative-config.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/manage-kubernetes-objects/kustomization.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md 1 file changed, 6 insertions(+), 8 deletions(-)
1. [ ] content/en/docs/tasks/tools/_index.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/tools/install-kubectl.md 1 file changed, 17 insertions(+), 5 deletions(-)
1. [ ] content/en/docs/tutorials/_index.md 1 file changed, 10 deletions(-)
1. [ ] content/en/docs/tutorials/hello-minikube.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tutorials/kubernetes-basics/explore/explore-intro.html 1 file changed, 3 insertions(+), 3 deletions(-)
1. [ ] content/en/docs/tutorials/services/source-ip.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/tutorials/stateful-application/cassandra.md 1 file changed, 26 insertions(+), 11 deletions(-)
1. [ ] content/en/docs/tutorials/stateful-application/zookeeper.md 1 file changed, 23 insertions(+), 18 deletions(-)
1. [ ] content/en/docs/tutorials/stateless-application/guestbook.md 1 file changed, 1 insertion(+), 1 deletion(-)

### 1 files to be deleted 
1. [ ] content/en/docs/reference/setup-tools/kubeadm/kubeadm.md

## Proposed Solution

Use `git diff` to check what is changed in the upstream. And apply the upstream changes manually 
to the `content/ko` of `dev-1.18-ko.11` branch.

For example:
```
# checkout `dev-1.18-ko.11`
...
# check what is updated in the upstream 
git diff dev-1.18-ko.10 dev-1.18-ko.11 -- content/en/_index.html
# apply changes to content/ko
vi content/ko/_index.html
...
# commit and push
...
# make PR to `dev-1.18-ko.11`
```

## Pages to Update

/language ko

### PR DESCRIPTION
**Pull request by pjhwa**
Closes #23532

Update outdated files in the dev-1.18-ko.11 branch.

63 files to be modified
- [x] content/en/docs/_index.md 1 file changed, 1 insertion(+)
- [x] content/en/docs/concepts/architecture/controller.md 1 file changed, 4 insertions(+), 5 deletions(-)
- [x] content/en/docs/concepts/cluster-administration/addons.md 1 file changed, 1 insertion(+), 3 deletions(-)
- [x] content/en/docs/concepts/cluster-administration/cloud-providers.md 1 file changed, 5 insertions(+), 76 deletions(-)
- [x] content/en/docs/concepts/configuration/manage-resources-containers.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/concepts/configuration/resource-bin-packing.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/configuration/secret.md 1 file changed, 4 insertions(+)
- [x] content/en/docs/concepts/containers/images.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/concepts/extend-kubernetes/extend-cluster.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/extend-kubernetes/operator.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/storage/persistent-volumes.md 1 file changed, 10 insertions(+)
- [x] content/en/docs/concepts/storage/volume-snapshots.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/concepts/workloads/controllers/daemonset.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/workloads/controllers/deployment.md 1 file changed, 7 insertions(+), 7 deletions(-)
- [x] content/en/docs/concepts/workloads/controllers/garbage-collection.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/workloads/controllers/job.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/workloads/controllers/replicaset.md 1 file changed, 4 insertions(+), 3 deletions(-)
- [x] content/en/docs/concepts/workloads/controllers/replicationcontroller.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/workloads/controllers/statefulset.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/workloads/pods/_index.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/workloads/pods/pod-lifecycle.md 1 file changed, 7 insertions(+), 8 deletions(-)
- [x] content/en/docs/reference/_index.md 1 file changed, 4 insertions(+), 2 deletions(-)
- [x] content/en/docs/reference/command-line-tools-reference/_index.md 1 file changed, 1 deletion(-)
- [x] content/en/docs/reference/command-line-tools-reference/feature-gates.md 1 file changed, 8 insertions(+), 5 deletions(-)
- [x] content/en/docs/reference/glossary/kube-apiserver.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/reference/glossary/managed-service.md 1 file changed, 6 insertions(+), 1 deletion(-)
- [x] content/en/docs/reference/issues-security/_index.md 1 file changed, 1 deletion(-)
- [x] content/en/docs/reference/kubectl/cheatsheet.md 1 file changed, 25 insertions(+), 28 deletions(-)
- [x] content/en/docs/reference/kubectl/overview.md 1 file changed, 18 insertions(+), 12 deletions(-)
- [x] content/en/docs/reference/setup-tools/_index.md 1 file changed, 1 deletion(-)
- [x] content/en/docs/reference/setup-tools/kubeadm/_index.md 1 file changed, 26 insertions(+), 1 deletion(-)
- [x] content/en/docs/reference/using-api/_index.md 1 file changed, 1 deletion(-)
- [x] content/en/docs/reference/using-api/client-libraries.md 1 file changed, 3 insertions(+), 3 deletions(-)
- [x] content/en/docs/setup/production-environment/container-runtimes.md 1 file changed, 82 insertions(+), 40 deletions(-)
- [x] content/en/docs/tasks/_index.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tasks/access-application-cluster/access-cluster.md 1 file changed, 9 insertions(+), 9 deletions(-)
- [x] content/en/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md 1 file changed, 12 insertions(+), 26 deletions(-)
- [x] content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md 1 file changed, 114 insertions(+), 38 deletions(-)
- [x] content/en/docs/tasks/administer-cluster/access-cluster-api.md 1 file changed, 7 insertions(+), 10 deletions(-)
- [x] content/en/docs/tasks/administer-cluster/access-cluster-services.md 1 file changed, 3 insertions(+), 2 deletions(-)
- [x] content/en/docs/tasks/administer-cluster/cluster-management.md 1 file changed, 18 insertions(+), 10 deletions(-)
- [x] content/en/docs/tasks/administer-cluster/dns-custom-nameservers.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md 1 file changed, 1 insertion(+), 5 deletions(-)
- [x] content/en/docs/tasks/administer-cluster/network-policy-provider/romana-network-policy.md 1 file changed, 4 insertions(+), 12 deletions(-)
- [x] content/en/docs/tasks/administer-cluster/network-policy-provider/weave-network-policy.md 1 file changed, 10 insertions(+), 14 deletions(-)
- [x] content/en/docs/tasks/configure-pod-container/quality-service-pod.md 1 file changed, 6 insertions(+), 2 deletions(-)
- [x] content/en/docs/tasks/debug-application-cluster/debug-init-containers.md 1 file changed, 3 insertions(+), 7 deletions(-)
- [x] content/en/docs/tasks/debug-application-cluster/logging-elasticsearch-kibana.md 1 file changed, 4 insertions(+), 3 deletions(-)
- [x] content/en/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md 1 file changed, 16 insertions(+), 8 deletions(-)
- [x] content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md 1 file changed, 9 insertions(+), 6 deletions(-)
- [x] content/en/docs/tasks/inject-data-application/define-environment-variable-container.md 1 file changed, 1 insertion(+), 11 deletions(-)
- [x] content/en/docs/tasks/manage-daemon/update-daemon-set.md 1 file changed, 1 insertion(+), 8 deletions(-)
- [x] content/en/docs/tasks/manage-kubernetes-objects/declarative-config.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tasks/manage-kubernetes-objects/imperative-command.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tasks/manage-kubernetes-objects/imperative-config.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tasks/manage-kubernetes-objects/kustomization.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md 1 file changed, 6 insertions(+), 8 deletions(-)
- [x] content/en/docs/tasks/tools/install-kubectl.md 1 file changed, 17 insertions(+), 5 deletions(-)
- [x] content/en/docs/tutorials/hello-minikube.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tutorials/kubernetes-basics/explore/explore-intro.html 1 file changed, 3 insertions(+), 3 deletions(-)
- [x] content/en/docs/tutorials/services/source-ip.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/tutorials/stateful-application/cassandra.md 1 file changed, 26 insertions(+), 11 deletions(-)
- [x] content/en/docs/tutorials/stateful-application/zookeeper.md 1 file changed, 23 insertions(+), 18 deletions(-)

1 files to be deleted
- [x] content/en/docs/reference/setup-tools/kubeadm/kubeadm.md

7 files already reflected
- [x] content/en/_index.html 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/architecture/nodes.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/containers/container-lifecycle-hooks.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/reference/using-api/api-overview.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tasks/tools/_index.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tutorials/_index.md 1 file changed, 10 deletions(-)
- [x] content/en/docs/tutorials/stateless-application/guestbook.md 1 file changed, 1 insertion(+), 1 deletion(-)

### CODE REVIEW

**Comment by pjhwa** [comment]
netlify 단계에서 실패해서 
Docy 관련 업데이트를 수행했더니 package.json, package-lock.json 파일이 업데이트되었습니다. 
참고하세요.

**Comment by gochist** [comment]
@seokho-son 
dev-1.18-ko.11 이 분기 된 이후에 release-1.18 에 추가 변경이 조금 있었는데, 이 내용을 fast forward merge 로 업데이트 하면 빌드 실패 문제가 해결될 것 같습니다. 

@pjhwa 
빌드 실패 관련 변경은 본 PR에서 제외하는 것이 좋을 것 같습니다. @kubernetes/sig-docs-ko-owners 에게 승인 권한이 없는 위치의 파일이기도 합니다.

/hold

**Comment by pjhwa** [comment]
> @pjhwa
> 빌드 실패 관련 변경은 본 PR에서 제외하는 것이 좋을 것 같습니다. @kubernetes/sig-docs-ko-owners 에게 승인 권한이 없는 위치의 파일이기도 합니다.

@gochist
네, 안그래도 그 부분이 마음에 걸렸는데, 일단 빌드가 실패하더라도 변경된 부분은 제외하겠습니다.

**Comment by seokho-son** [comment]
@ClaudiaJKang 혹시 리뷰 진행 중이신가요? 아직 작업 진행 많이 안하셨으면 제가 맡아서 할 수 있을 것 같습니다.. ^^
(이번 주 중에는 release-1.18 작업을 마무리하고자 합니다.)

**Comment by pjhwa** [comment]
Update outdated files in the dev-1.19-ko.1 branch PR( #23702 )에서 @seokho-son 님께서 리뷰해주신 내용을 동일하게 반영하였습니다. 참고하세요~

- https://github.com/kubernetes/website/pull/23702#discussion_r484153109
- https://github.com/kubernetes/website/pull/23702#discussion_r484155521
- https://github.com/kubernetes/website/pull/23702#discussion_r484155781
- https://github.com/kubernetes/website/pull/23702#discussion_r484156195
- https://github.com/kubernetes/website/pull/23702#discussion_r484156873

File: content/ko/docs/concepts/cluster-administration/cloud-providers.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ weight: 30
 
 <!-- overview -->
 이 페이지에서는 특정 클라우드 제공자에서 실행 중인 쿠버네티스를 관리하는 방법에
-대해 설명한다.
+대해 설명한다. 다른 많은 타사 클라우드 제공자 프로젝트가 있지만, 이 목록은 쿠버네티스 자체에 의존하거나, 포함되어있는 프로젝트에 한정한다.
```
Comment (seokho-son) [inline content/ko/docs/concepts/cluster-administration/cloud-providers.md:9]:
```suggestion
대해 설명한다. 다른 클라우드 제공자의 프로젝트가 많이 있지만, 여기의 목록은 쿠버네티스 자체에 의존하거나, 포함되어있는 프로젝트로 한정한다.
```
조금 자연스럽게 수정해보았습니다.

File: content/ko/docs/concepts/workloads/controllers/replicationcontroller.md [hunk 2]
```diff
@@ -112,7 +112,7 @@ nginx-3ntk0 nginx-4ok8v nginx-qrm3m
 다른 모든 쿠버네티스 컨피그와 마찬가지로 레플리케이션 컨트롤러는 `apiVersion`, `kind`, `metadata` 와 같은 필드가 필요하다.
 레플리케이션 컨트롤러 오브젝트의 이름은 유효한
 [DNS 서브도메인 이름](/ko/docs/concepts/overview/working-with-objects/names/#dns-서브도메인-이름)이어야 한다.
-컨피그 파일의 동작에 관련된 일반적인 정보는 다음을 참조하라 [쿠버네티스 오브젝트 관리 ](/ko/docs/concepts/overview/working-with-objects/object-management/).
+컨피그 파일의 동작에 관련된 일반적인 정보는 [쿠버네티스 오브젝트 관리 ](/ko/docs/concepts/overview/working-with-objects/object-management/)를 참고한다.
```
Comment (seokho-son) [inline content/ko/docs/concepts/workloads/controllers/replicationcontroller.md:115]:
```suggestion
컨피그 파일의 동작에 관련된 일반적인 정보는 [쿠버네티스 오브젝트 관리](/ko/docs/concepts/overview/working-with-objects/object-management/)를 참고한다.
```
공백이 포함되어 있네요^^

File: content/ko/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md [hunk 3]
```diff
@@ -127,20 +123,14 @@ Debian 컨테이너에서 nginx 웹 서버가 호스팅하는 문서의 루트 
...
-더 공부한다.
+* [합성 컨테이너(composite container) 패턴](https://kubernetes.io/blog/2015/06/the-distributed-system-toolkit-patterns)에 관하여 더 공부한다.
 
-* [모듈 구조를 위한 합성 컨테이너 구조](http://www.slideshare.net/Docker/slideshare-burns)에 관하여
-더 공부한다.
+* [모듈 구조를 위한 합성 컨테이너 구조](https://www.slideshare.net/Docker/slideshare-burns)에 관하여 더 공부한다.
 
-* [파드에서 저장소로 볼룸을 사용하도록 구성하기](/ko/docs/tasks/configure-pod-container/configure-volume-storage/)에 관하여
-확인한다.
+* [파드에서 저장소로 볼룸을 사용하도록 구성하기](/ko/docs/tasks/configure-pod-container/configure-volume-storage/)에 관하여 확인한다.
```
Comment (seokho-son) [inline content/ko/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md:133]:
```suggestion
* [파드에서 저장소로 볼륨을 사용하도록 구성하기](/ko/docs/tasks/configure-pod-container/configure-volume-storage/)에 관하여 확인한다.
```

**Review by seokho-son — COMMENTED** [review]
@pjhwa 컨트리뷰션 감사합니다 ^^
리뷰 완료하였습니다. 마이너 의견 있어서 공유 드립니다.

File: content/ko/docs/concepts/cluster-administration/cloud-providers.md (hunk 1 above)
Comment (pjhwa) [reply content/ko/docs/concepts/cluster-administration/cloud-providers.md:9]: 반영하겠습니다.

File: content/ko/docs/concepts/workloads/controllers/replicationcontroller.md (hunk 2 above)
Comment (pjhwa) [reply content/ko/docs/concepts/workloads/controllers/replicationcontroller.md:115]: 반영하겠습니다.

File: content/ko/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md (hunk 3 above)
Comment (pjhwa) [reply content/ko/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume.md:133]: 반영하겠습니다.

**Comment by pjhwa** [comment]
@seokho-son 리뷰 감사합니다!
의견주신 내용은 모두 반영하였습니다. ^^

**Comment by seokho-son** [comment]
@pjhwa 확인 완료하였습니다! 

이로써, 1.18의 작업이 모두 완료되었네요! 감사합니다.

/lgtm
/approve

**Comment by seokho-son** [comment]
@pjhwa 

package-lock.json 와
package.json 이 추가되어 있었네요. 

관련해서는 한글 컨텐츠를 위해서만 변경할 수는 없고, 승인 권한도 없으므로, 제외하는 것이 맞을 것 같습니다.
혹시, 제외하면 netlify 빌드 테스트가 실패하는 상태인가요?

**Comment by pjhwa** [comment]
@seokho-son 로컬에서 hugo로 확인하는 과정에서 들어간 것 같습니다.
지난 번에도 빼고 올려도 되더라고요. 다시 빼놓겠습니다.

**Comment by seokho-son** [comment]
@pjhwa 감사합니다. 스쿼싱도 부탁드릴게요^^

**Comment by pjhwa** [comment]
> @pjhwa 감사합니다. 스쿼싱도 부탁드릴게요^^

푸시하고 나서 바로 approve 레이블이 붙어서, 깜빡했습니다. ^^

**Pull request merged**